src/storage.rs

```diff
@@ -25,6 +25,7 @@ pub enum EthConnectorStorageId {
     Contract = 0x0,
     FungibleToken = 0x1,
     UsedEvent = 0x2,
+    Statistic = 0x3,
```

Comment:
At the moment this is used only to represent the number of unique accounts, but the key used is too general.
Let's create another subkey for the `Statistics` items. I believe we agreed to have only distinct accounts counter so far. But still creating a subkey could be useful for the future and simplify the possible storage migration needed.

If we don't want to add the subkey, let's call it at least explicit:
`    StatisticsNumDistinctAuroraAccounts = 0x3,`